scalar.c

```diff
@@ -230,6 +230,31 @@ static int set_recommended_config(int reconfigure)
 				     fsmonitor.key, fsmonitor.value);
 	}
 
+	/*
+	 * Set HTTP/1.1 for Azure DevOps URLs
+	 */
+	if (!git_config_get_string("remote.origin.url", &value)) {
+		if (strstr(value, "dev.azure.com") || 
+		    strstr(value, "visualstudio.com")) {
```

Comment:
Please be more stringent! Here, we could at least test for `.visualstudio.com/`. It would probably be overkill to do an entire regex matching (what with all the boilerplate of `regcomp()`/`regexec()`/`regfree()` required), so let's avoid that and just state that `.visualstudio.com/` is good enough and live with the potential false-positive if anybody creates a `x.visualstudio.com` org on GitHub.

A commit message would be a really good place to document this. Please compose one, marking it up as `amend!` so that the end result is easily rebased.